ext/search_kit/search_kit.php

```diff
@@ -24,6 +24,16 @@ function search_kit_civicrm_permission(&$permissions) {
     'description' => E::ts('Gives non-admin users access to the SearchKit UI to create, update and delete searches and displays'),
     'implied_by' => ['administer CiviCRM data'],
   ];
+  $permissions['edit own search_kit'] = [
+    'label' => E::ts('SearchKit: edit searches that the user created'),
+    'description' => E::ts('Gives non-admin users the permission to edit their own searches and displays'),
+    'implied_by' => ['administer CiviCRM data'],
```

Comment:
I think this should be 
```suggestion
    'implied_by' => ['administer search_kit'],
```